frontend/src/components/login/login.tsx

```diff
@@ -1,3 +1,94 @@
-const Login: React.FC = () => <h1>Login</h1>;
+import { AppRoute, UserPayloadKey } from 'common/enums/enums';
+import FormInput from 'components/common/form-input/form-input';
+import { useAppDispatch, useAppForm, useAppSelector, useCallback, useNavigate } from 'hooks/hooks';
+import { Button, Container, FloatingLabel, Form } from 'react-bootstrap';
+import { Google } from 'react-bootstrap-icons';
+import { NotificationManager } from 'react-notifications';
+import { loginUser } from 'store/auth/actions';
+import { login as loginValidationSchema } from 'validation-schemas/validation-schemas';
+import { DEFAULT_LOGIN_PAYLOAD } from './common/constants';
+import './styles.scss';
+
+const Login: React.FC = () => {
+  const navigate = useNavigate();
+  const dispatch = useAppDispatch();
+  const isLoading = useAppSelector(state => state.auth.isLoading);
+
+  const { control, errors, handleSubmit } = useAppForm({
+    defaultValues: DEFAULT_LOGIN_PAYLOAD,
+    validationSchema: loginValidationSchema,
+  });
+
+  const handleLogin = useCallback(
+    loginPayload => dispatch(loginUser(loginPayload)),
+    [dispatch],
+  );
+
+  const onLogin = (values: object): void => {
+    handleLogin(values)
+      .unwrap()
+      .then(() => {
+        navigate(AppRoute.ROOT);
+      })
+      .catch((err: Error) => {
+        NotificationManager.error(err.message);
+      });
+  };
+
+  return (
+    <Container>
+      <Form className="auth-form w-100" onSubmit={handleSubmit(onLogin)}>
+        <p className="fs-1 text-center mb-4">Sign in</p>
+
+        <fieldset disabled={isLoading}>
+          <FloatingLabel
+            controlId="authEmail"
+            label="Email address"
+            className="mb-3"
+          >
+            <FormInput
+              name={UserPayloadKey.EMAIL}
+              control={control}
+              errors={errors}
+              type="email"
+              placeholder="Email address"
+            />
+          </FloatingLabel>
+
+          <FloatingLabel
+            controlId="authPassword"
+            label="Password"
+            className="mb-3"
+          >
+            <FormInput
+              name={UserPayloadKey.PASSWORD}
+              control={control}
+              errors={errors}
+              type="password"
+              placeholder="Password"
+            />
+          </FloatingLabel>
+
+          <Form.Group className="auth-form__checkbox-container mb-4" controlId="authCheckbox">
+            <Form.Check type="checkbox" label="Remember me" />
+            <p className="auth-form__btn-link">
+              <a className="auth-form__link" href="#">Forgot password?</a>
+            </p>
+          </Form.Group>
+
+          <div className="d-grid gap-2">
+            <Button variant="danger" type="submit" size="lg">
```

Comment:
For complex components use react-bootstrap.
But for button better to use just html element